docs/table/access_table.rst

```diff
@@ -316,16 +317,22 @@ structured array by creating a copy or reference with ``np.array``::
 
 Table Equality
 --------------
+
 We can check table data equality using two different methods:
 
-- The ``==`` comparison operator.  This returns a ``True`` or ``False`` for
-  each row if the *entire row* matches.  This is the same as the behavior of
-  numpy structured arrays.
+- The ``==`` comparison operator. This returns a ``True`` or ``False`` for
+  each row if the *entire row* matches. This is the same as the behavior of
+  ``numpy`` structured arrays.
```

Comment:
Should this be `NumPy`?

Reply:
Right now our style convention is to do lowercase and double backticks for a proper noun like NumPy and Astropy when the text is referring to the core package or code, and uppercase when referring to "The Project." Do structured arrays refer to NumPy code or a package?

Reply:
Structure arrays would be code or a package, not the NumPy "Project".